packaging/Dockerfile.sitl_gzsim

```diff
@@ -44,9 +44,10 @@ RUN DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt install -y \
 	fastddsgen
 
 # Checkout the px4 version and build it
-RUN git clone https://github.com/tiiuae/px4-firmware
+COPY px4-firmware px4-firmware
+#RUN git clone --branch pr-gzsim-sitl-harmonic-support https://github.com/tiiuae/px4-firmware
```

Comment:
maybe this can be removed? Of course no harm done keeping it here if you like to have it as a reference

Reply:
yeah, will clean it up